prompting/tools/datasets/generic_instruction.py

```diff
@@ -4,7 +4,7 @@
 class GenericInstructionDataset(TemplateDataset):
     "Generic question dataset, which creates LLM prompts for asking questions."
     name = "generic_instruction"
-    query_template = (
+    query_template : str = (
```

Comment:
PEP8 spacing conventions

```suggestion
    query_template: str = (
```